examples/diffusion/textual_inversion_stable_diffusion/README.md

```diff
@@ -0,0 +1,235 @@
+# Textual Inversion with Stable Diffusion
+
+This example demonstrates how to incorporate your own images into AI-generated art via
+[Textual Inversion](https://textual-inversion.github.io).
+
+The development of [Latent Diffusive Models](https://arxiv.org/abs/2112.10752) has made
+it possible to run (and fine-tune) diffusion-based models on consumer-grade GPUs. Such tasks are
+made even easier by the release
+of [Stable Diffusion](https://stability.ai/blog/stable-diffusion-announcement) and the
+development of the 🤗 [Huggingface 🧨 Diffusers](https://huggingface.co/docs/diffusers/index) library.
+
+The present code uses Determined's Core API to seamlessly incorporate 🧨 Diffusers
+and 🚀 [Accelerate](https://huggingface.co/docs/transformers/accelerate) into the
+Determined framework.
+
+## Before You Start: 🤗 Account, Access Token, and License
+
+In order to use this repository's implementation of Stable Diffusion, you must:
+
+* Have a [Huggingface account](https://huggingface.co/join).
+* Have a [Huggingface User Access Token](https://huggingface.co/docs/hub/security-tokens).
+* Accept the Stable Diffusion license (click on _Access
+  repository_  [in this link)](https://huggingface.co/CompVis/stable-diffusion-v1-4).
+
+## Walkthrough: Basic Usage
+
+Below we walk through the Textual Inversion workflow, which consists of two stages:
+
+1. Fine-tune Stable Diffusion on a set of
+   user-provided training images featuring a new concept
+2. Incorporate representations of the
+   concept into generated art.
+
+### Fine-Tuning
+
+After including your user access token in the `finetune_const.yaml` config file by
+replacing `YOUR_HF_AUTH_TOKEN_HERE` where it reads
+
+```yaml
+environment:
+  environment_variables:
+    - HF_AUTH_TOKEN=YOUR_HF_AUTH_TOKEN_HERE
+```
+
+a ready-to-go fine-tuning experiment can be run by executing the following in the present directory:
+
+```bash
+det experiment create finetune_const.yaml .
+```
+
+This will submit an experiment which introduces a new embedding vector into the world of Stable
+Diffusion which we will fine-tune to correspond to the concept of the Determined AI logo, as
+represented
+through training images found in `det_logos/`, such as the example found below (placed on a
+background for
+improved results):
+
+![det-logo](./det_logos/on_a_dark_blue_oil_painting_of_ocean_waves.jpg)
+
+A corresponding concept token, chosen to be `<det-logo>` as specified in the `concept_tokens` field
```

Comment:
They don't, the concepts should just be strings that don't already exist in the tokenizer, which the <> ensure.

This plus some of the other issues you flagged (which have also caused others confusion) came from the [Hugging Face example](https://huggingface.co/docs/diffusers/v0.4.1/en/training/text_inversion), which I was initially adhering to much more closely.  Going to remove the <> and related issues from HF, since they have come up repeatedly, thanks!

Reply:
And actually, it's alright if the concepts do exist in the tokenizer, practically speaking, they'll just be replaced by dummy strings before being processed by the tokenizer anyway.